src/presentation/webapp/core/pages/manual-sync/ManualSyncPage.tsx

```diff
@@ -160,7 +160,6 @@ const ManualSyncPage: React.FC = () => {
 
     const handleSynchronization = async (syncRule: SynchronizationRule) => {
         loading.show(true, i18n.t(`Synchronizing ${syncRule.type}`));
-
         const result = await compositionRoot.sync.prepare(syncRule.type, syncRule.toBuilder());
```

Comment:
Do not remove newline:

```suggestion

        const result = await compositionRoot.sync.prepare(syncRule.type, syncRule.toBuilder());
```